packages/thumbprint-react/components/Fab/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import classNames from 'classnames';
+import noop from 'lodash/noop';
+import styles from './index.module.scss';
+
+interface TextFabProps {
+    /**
+     * Children content to render.
+     */
+    children: React.ReactNode;
+    /**
+     * Visually highlights the fab if it‘s active.
+     */
+    active?: boolean;
+    /**
+     * Function that will run when the fab is clicked on.
+     */
+    onClick?: () => void;
+}
+
+export function TextFab({ children, active = false, onClick = noop }: TextFabProps): JSX.Element {
+    return (
+        <button
+            className={classNames(styles.root, styles.textFab, {
+                [styles.active]: active,
+                [styles.inactive]: !active,
+            })}
+            type="button"
+            onClick={onClick}
+        >
+            {children}
+        </button>
+    );
+}
+
+interface IconFabProps {
+    /**
+     * Icon from [Thumbprint Icons](/icons/) to render within the button. It must be one of the
+     * `small` icons.
+     */
+    icon: React.ReactNode;
+    /**
+     * Visually highlights the fab if it‘s active.
+     */
+    active?: boolean;
+    /**
+     * Function that will run when the fab is clicked on.
+     */
+    onClick?: () => void;
+}
+
+export function IconFab({ icon, active = false, onClick = noop }: IconFabProps): JSX.Element {
```

Comment:
To make this accessible to screen readers, we need to add a prop that adds a `aria-label` to the `<button>`. We've used a prop called `accessibilityLabel` in other components.